src/Microsoft.SqlTools.ServiceLayer/Metadata/SmoScripterFactory.cs

```diff
@@ -0,0 +1,191 @@
+                DriAllKeys = false,
+                DriChecks = false,
+                DriClustered = false,
+                DriDefaults = false,
+                DriForeignKeys = false,
+                DriIncludeSystemNames = false,
+                DriIndexes = false,
+                DriNonClustered = false,
+                DriPrimaryKey = false,
+                DriUniqueKeys = false,
+                DriWithNoCheck = false,
+                EnforceScriptingOptions = true,
+                ExtendedProperties = false,
+                FullTextCatalogs = false,
+                FullTextIndexes = false,
+                FullTextStopLists = false,
+                IncludeDatabaseContext = false,
+                IncludeDatabaseRoleMemberships = false,
+                IncludeFullTextCatalogRootPath = false,
+                IncludeHeaders = false,
+                IncludeIfNotExists = false,
+                IncludeScriptingParametersHeader = false,
+                Indexes = false,
+                LoginSid = false,
+                NoAssemblies = true,
+                NoCollation = true,
+                NoCommandTerminator = true,
+                NoExecuteAs = true,
+                NoFileGroup = true,
+                NoFileStream = true,
+                NoFileStreamColumn = true,
+                NoIdentities = true,
+                NoIndexPartitioningSchemes = true,
+                NoMailProfileAccounts = true,
+                NoMailProfilePrincipals = true,
+                NonClusteredIndexes = false,
+                NoTablePartitioningSchemes = true,
+                NoVardecimal = false,
+                NoViewColumns = false,
+                NoXmlNamespaces = false,
+                OptimizerData = false,
+                Permissions = false,
+                PrimaryObject = true,
+                SchemaQualify = true,
+                SchemaQualifyForeignKeysReferences = true,
+                ScriptBatchTerminator = false,
+                ScriptData = false,
+                ScriptDataCompression = false,
+                ScriptDrops = false,
+                ScriptForAlter = false,
+                ScriptForCreateDrop = false,
+                ScriptForCreateOrAlter = true,
+                ScriptOwner = false,
+                ScriptSchema = true,
+                ScriptXmlCompression = false,
+                SpatialIndexes = false,
+                Statistics = false,
+                TimestampToBinary = false,
+                ToFileOnly = false,
+                Triggers = false,
+                WithDependencies = false,
+                XmlIndexes = false                
+            };
+
+            var scripter = new Scripter(server);
+            scripter.Options = scriptingOptions;
+            var generatedScripts = scripter.Script(urns);
+
+            var scripts = new List<string>();
+            foreach (var s in generatedScripts)
+            {
+                if (s.StartsWith("CREATE TABLE"))
```

Comment:
It's including things like `SET ANSI_NULLS ON` and `SET QUOTED_IDENTIFIER ON` for every create table script. We don't need any of that since the only thing needed is the table name and column names and types.

Reply:
Is there any harm in having it?

If you're going to make this into a very generic sounding service then I'd say let's try to avoid doing extra unexpected things like removing parts of the generated data. Otherwise if someone else tries to use this and doesn't realize it's been made specifically for copilot use that could be problematic.

Reply:
Also keep in mind there's other statements that can be applied, such as `ALTER TABLE...ADD PRIMARY KEY` which would probably be useful for copilot to know about.